### ISSUE TITLE
Problem with font and line rendering

### ISSUE THREAD

**Issue by iyefrat** [polybar/polybar#1749]
## The Problem:
It seems that in some cases, the line that is drawn on the top and bottom doesn't completely cover some glyphs. This is best illustrated with the following screenshot generated from the config below: 

![Screenshot-2019-04-22-13-53-08](https://user-images.githubusercontent.com/19792685/56497482-57dfba80-6506-11e9-9e25-4030f50fd9f2.png)

As you can see (probably only after opening the picture in fullscreen), in the right powerline symbol, the leftmost side of it is not covered by the line, even though everything else is.

```dosini

;Example config

[colors]
background = #ffffff
background-light = #3b4252
background-alt = #144133
foreground = #d7ffd7
foreground-alt = #88c0d0
highlight = #39c6fe
secondary = #e60053
alert = #bf616a
alert2 = #ebcb8b

;==============================
;		BARS
;==============================

[bar/example]
; comment line below if you only have 1 screen
monitor = ${env:MONITOR:HDMI-1}
;monitor = ${env:MONITOR:eDP-1}

width = 100%
height = 32
fixed-center = true

background = ${colors.background}
foreground = ${colors.foreground}
line-size = 4
line-color = ${colors.highlight}

font-0 = "Fura Code Nerd Font:size=22;3"
font-1 = "Misc Termsyn:size=12;2"

; desktop PC modules
modules-center = xwindow

;==============================
;		MODULES
;==============================
[module/xwindow]
type = internal/xwindow

format = <label>
label = %{F#144133}%{R F-}%{T2}%title%%{R B-}
;label-maxlen = 75

format-foreground = ${colors.foreground}
format-overline = ${colors.background}
format-underline = ${colors.background}
```
The polybar log doesn't output anything.

## Environment:
* WM: i3
* Distro: Arch Linux
* Output of `polybar -vvv`: 
```
polybar 3.3.1

Features: +alsa +curl +i3 +mpd +network(libnl) +pulseaudio +xkeyboard

X extensions: +randr (+monitors) -render -damage -sync +composite +xkb +xrm +xcursor

Build type: RelWithDebInfo
Compiler: /usr/bin/c++
Compiler flags: -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -Wall -Wextra -Werror -Wno-noexcept-type -pedantic -pedantic-errors
Linker flags: -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now
```
Thank you for your hard work!

**Comment by patrick96** [comment]
Hmm. I was able to reproduce this kind of (that vertical line is on the left powerline glyph for me). Don't really know what causes it, but was able to fix it by disabling anti-aliasing for the powerline font. Could you also try that?

**Comment by seamus-45** [comment]
Also have this problem with the `Droid Sans Mono` family fonts. 
Option `:antialias=false` does not help
With `Droid Sans` font problem is gone.

**Comment by patrick96** [comment]
Should be fixed by #2802
